### ISSUE TITLE
Not working on Mojave

### ISSUE THREAD

**Issue by mrchief** [2ndalpha/gasmask#166]
It seems that it no longer updates `/etc/hosts` on Mac OS Mojave. I'm running the latest version and tried it a few times, including restarting the app and even Macbook. 

There are no errors but switching hosts don't update `/etc/hosts`.

**Comment by phoolish** [comment]
It updates the `/etc/hosts` file on my system, but mojave seems to continue to ignore the change.

**Comment by mrchief** [comment]
GasMask doesn't update the file on my system though. 

I tried bitbar's host file switcher [plugin](https://getbitbar.com/plugins/System) and that updated hosts file correctly. Not as nice as GasMask but at least I have a way of swtiching hosts for now.

![image](https://user-images.githubusercontent.com/781818/53926778-c2ba5c80-4052-11e9-9f67-567c9b53ebe3.png)

Also, Mojave needs the `sudo killall -HUP mDNSResponder` to flush DNS cache after host change.

**Comment by MadMacMad** [comment]
it doesn't change the host file...
but it blocks all outgoing ip's
try ping a blocked ip
works on Mojave

**Comment by mrchief** [comment]
> it doesn't change the host file...
but it blocks all outgoing ip's in your gas mask list

What's the secret sauce?

**Comment by ccharlton** [comment]
Did you try the latest pre-stable version?

**Comment by mrchief** [comment]
> Did you try the latest pre-stable version?

No, I just switch between local and remote files a few times, kill the mdns thingy and restart browser. If it still doesn't work, I try the bitbar scripts. 

Is the pre-stable stable enough for regular use?

**Comment by sojuzpl** [comment]
on my Mojave app works without problems - I'm using 0.8.6

**Comment by phoolish** [comment]
Seems to be solved with 0.8.6. Thanks!

**Comment by mrchief** [comment]
Actually, I'm on 0.8.6 - I'm not sure if it gets auto-updated. If not, then I've had that when I opened this issue.

**Comment by MadMacMad** [comment]
i would really like to see a 0.8.7 or a 0.9.0 version with dark mode and the https/ssl feature working!
also using 0.8.6 on Mojave... but sometimes the pref window crashes... and some other glitches...
if someone can make it .. plz and let us know...
thx

**Comment by ccharlton** [comment]
"I'm Spartacus!"... oh, I mean, I too am on version 0.8.6 w/Mojave and doing fine.

**Comment by mrchief** [comment]
I haven't seen any issues so far with 0.8.6

**Comment by mrchief** [comment]
Looks like I spoke too soon - I had the issue today. Switching between original and Combined files fixed it though.

**Comment by dmcourtn** [comment]
I am getting an error from Gas Mask 0.8.6 when I try to "Create from URL"

> "Bad Content. Can't download the hosts file. It contains unknown content." 

The URL I'm linking to is Steven Black's: 

> https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/fakenews-gambling-porn/hosts

I'm running a 2018 MacBook Pro with MacOS Mojave 10.14.5 (18F203). 

Screenshot attached. 

Best,
Dale
<img width="559" alt="Screen Shot 2019-07-12 at 13 13 23" src="https://user-images.githubusercontent.com/21040897/61155915-fac12880-a4a6-11e9-9fb4-d52831fdbcb9.png">

**Comment by nucleare2** [comment]
Not starting up on on 10.14.6. Seems to start, but GUI does not appear and icon not on Dock or cmd-tab.

**Comment by webuxr** [comment]
Also using Mojave 10.14.6 and Gas Mask 0.8.6 and everything works fine except 1 annoying UI-related issue in the Editor pane.

At the top of the editor pane (in the pink-colored area), when looking at a combined host, the list of selected host files, my cursor cannot touch the bottom-most host file. Screenshot attached.

This is problematic as I cannot remove the last, bottom-most host file from my combined host file. I can edit the individual file (that's in the last position in the combined host file) by selecting it in the left sidebar and editing as usual. The only way I've found to remove the bottom-most host file from my combined host is to remove/delete it from the left sidebar.

As shown in the screenshot, I've taken to creating a dummy "blank" host file that I've placed as the bottom-most host file.

I'm not sure if this issue warrants a new ticket but I can certainly create one if requested.

I've been using Gas Mask for several years and truly enjoy the flexibility it provides. I'd love to see a new version of the app to address this soon.

Thank you!

<img width="689" alt="Screen Shot 2020-04-03 at 9 36 21 AM" src="https://user-images.githubusercontent.com/1559023/78373134-cab73980-758f-11ea-8faf-5c09de3beea4.png">

**Comment by xthursdayx** [comment]
> I am getting an error from Gas Mask 0.8.6 when I try to "Create from URL"
> 
> > "Bad Content. Can't download the hosts file. It contains unknown content."
> 
> The URL I'm linking to is Steven Black's:
> 
> > https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/fakenews-gambling-porn/hosts
> 
> I'm running a 2018 MacBook Pro with MacOS Mojave 10.14.5 (18F203).
> 
> Screenshot attached.
> 
> Best,
> Dale
> <img alt="Screen Shot 2019-07-12 at 13 13 23" width="559" src="https://user-images.githubusercontent.com/21040897/61155915-fac12880-a4a6-11e9-9fb4-d52831fdbcb9.png">

I'm having this same problem with another StevenBlack list. Any idea what th issue is?
